src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable font-family-name-quotes */
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.countainer{
+  width: 1200px;
+  display: flex;
+}
```

Comment:
```suggestion
```
remove flex from body and redundant container shouldn't fail tests. header should be on full width of viewport.